snuba/datasets/tags_column_processor.py

```diff
@@ -132,20 +277,34 @@ def __tags_expr(
         cols_used = query.get_all_referenced_columns() & set(
             [qualified_key, qualified_value]
         )
+
+        filter_tags = ",".join([f"'{tag}'" for tag in self.__get_filter_tags(query)])
```

Comment:
I cannot fit this logic in a query processor just yet, because tags are processed here (this happens during formatting still) and I cannot properly apply aliases to the old system from the query processor.